server/src/app.controller.ts

```diff
@@ -1,28 +1,57 @@
-import { Controller, Get } from '@nestjs/common';
-import { TestService, AppService } from './app.service';
+import { Controller, Get, Post, Body } from '@nestjs/common';
+import { TestService, BrowseService } from './app.service';
 import ApiFactory from './api/import';
 
 
 function getApi(description: string) {
   return ApiFactory.getInstance().getApi('v1.0', description);
 }
+
+function processRequest(func: Function, body): string {
```

Comment:
typescript-eslint doesn't like non-typed parameters.

It really doesn't like ``Function`` as a type